packages/shared/utils/testing.ts

```diff
@@ -126,6 +128,7 @@ export function createPointDescription({
 }
 
 export type SourceTuple = ["Debugger.newSource", newSource];
+export type SourcesTuple = ["Debugger.newSources", newSources];
 
 export function createSource({
```

Comment:
@bvaughn This testing stuff is what I wanted to ask you about. I'm not familiar with this code, so while I've done my best to add all this in parallel with what was already here, I'm not sure if there's any issues we should worry about here for the test fixture data, and how we manage that as we swap the the API over to the new event type.

Reply:
As of #9107, there is no longer any fixture data being used by our screenshot tests.

Reply:
I don't think I have enough context to know how that relates to the question. Does that mean I'm fine to land this as-is?